test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -89,11 +89,14 @@ class SolidityExecutionFramework: public dev::test::ExecutionFramework
 
 	using rational = boost::rational<dev::bigint>;
 
-	static u256 createFixed(s256 _numerator, s256 _denominator = 1, unsigned int _decimalDigits = 0)
+	static u256 createFixed(s256 _numerator, s256 _denominator = 1, unsigned int _decimalDigits = 0, unsigned int _totalBits = 256)
 	{
+		solAssert(_totalBits <= 256, "Illegal _totalBits in createFixed");
```

Comment:
I think we're not using `solAssert`s in the test suite because it throws a solidity specific exception. This one should use boost exceptions, e.g. `BOOST_REQUIRE_MESSAGE(_totalBits <= 256, ...)`

Reply:
Okay, will update!